src/styles/colors.js

```diff
@@ -0,0 +1,61 @@
+const white = "#FFFFFF";
+
+const primaryText = "#1F1F1F";
+
+const primaryBackground = "#F7F8FC";
+
+const darkGradientBackground =
+  "linear-gradient(93.51deg, #182E7C 8.9%, #030D67 94.31%)";
+
+const lightGradientBackground =
+  "linear-gradient(93.51deg, #1782DC 8.9%, #6AB7F7 94.31%)";
+
+const activeButton = "#3678E3";
+
+const hoverButton = "#124CAB";
+
+const disabledButton = "#C9DAF6";
+
+const activeLink = "#3678E3";
+
+const placeholder = "#A8A8A8";
+
+const errorMessage = "#E12D43";
+
+const successMessage = "#058C23";
+
+const tagText = "#DA7602";
+
+const tagBackground = "#FFF1DE";
+
+const mainHeader = "#161C56";
+
+const primarySubtitle = "#124CAB";
+
+const seconderySubtitle = "#F88B0C";
+
+const copyrights = "#DCDCDC";
+
+const textDetails = "#757575";
```

Comment:
I'm not sure whether I like those names, can e.g. `tagBackground` be used in other place then in tag background? Also a question to other names.
Anyway, I am fine also to keep it this way and rename it once we use it in particular styles and it's easier to see the proper name for it. ;)

Reply:
🙆‍♀️